lib/externalApis/dashcore/rpc.js

```diff
@@ -43,6 +43,11 @@ const getBestBlockHeight = () => new Promise((resolve, reject) => {
   });
 });
 
+/**
+ * @param {string} hash
+ * @param {number} isParsed - can be 1 or 0
+ * @return {Promise<Object>}
+ */
```

Comment:
isParsed can be 0, 1 or 2
https://github.com/dashpay/dash/blob/master/src/rpc/blockchain.cpp#L847

Reply:
I've removed this file from the PR due to changes are not related to it.